src/main/java/sirius/db/text/BasicIndexTokenizer.java

```diff
@@ -17,8 +17,14 @@
 
     @Override
     protected ChainableTokenProcessor createProcessor() {
+        // ATTENTION: The pipeline used to contain a "new ReduceCharacterProcessor()" stage. This lead to the search
+        // index containing normalised strings, unlike earlier verions of sirius-db. Consequently, MEMOIO was no longer
```

Comment:
this wasn't just a MEMOIO problem - essentially the ElasticQueryCompiler doesn't perform the same reduction - therefore all terms with appropriate characters became problematic / unfindable / whatever :-)

Reply:
revised